src/app/app.cpp

```diff
@@ -353,6 +357,11 @@ int App::initialize(const AppOptions& options)
     // clipboard from the ui::Entry control.
     m_uiSystem->setClipboardDelegate(&m_modules->m_clipboard);
 
+    // Set the InputOptionsDelegate impl to communicate peripheral
+    // input options (as scroll wheel sensitivity) between
+    // Preferences variables and ui::Manager.
+    m_uiInputOptions->setDelegate(&m_modules->m_inputOptions);
```

Comment:
The input options should be a member of UISystem (there is no need to create another global object in the UI layer for this).

So here we could do something like:

m_uiSystem->setInputOptionsDelegate(...)